modules/python/pylib/syslogng/__init__.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+#############################################################################
```

Comment:
executable scripts contained this, whereas modules to be imported didn't.
having a shebang line indicates that the file would be executable, which
they are not.

-- 
Bazsi

On Mon, Mar 21, 2016 at 10:22 AM, Tamas Nagy <notifications@github.com>
wrote:

> In modules/python/pylib/syslogng/__init__.py
> <https://github.com/balabit/syslog-ng/pull/985#discussion_r56795686>:
>
> > @@ -0,0 +1,22 @@
> > +#!/usr/bin/env python
> > +#############################################################################
>
> @bazsi <https://github.com/bazsi>
> 1. Some of the Python files contained this, some of them did not. I wanted
> to make it uniform.
> 2. I recommend using a shebang in each and every script to make file magic
> consistent on all platforms.
>
> In light of these, do you agree?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/balabit/syslog-ng/pull/985/files/4bd9d6e8f9bcba316d6ae8b02b701ffa4d85f6ea#r56795686>
>